Notify/WEB/ClientApp/src/components/Notification/Notification.js

```diff
@@ -1,79 +1,39 @@
-﻿import React, { Component } from 'react';
+﻿import React from 'react';
+import './Notification.css'
 
-import NotificationService from '../../services/NotificationService';
 
 
-export default class Notification extends Component {
+export default function Notification( {notification = null, error = null } ){
 
-    notificationService = new NotificationService();
 
-    state = {
-        Id: null,
-        Title: null,
-        Body: null,
-        Icon: null,
-        Image: null,
-        NotificationActions: null
-    };
-
-    constructor(props) {
-        super(props);
+    function icon() {
```

Comment:
you should make your props flat
`function Notification({icon, image, title, body})`
and this component shouldn't be aware of errors, its responsibility is just to render notification. You should manage your error state in the component upper in hierarchy and render some error there.